src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -81,7 +81,11 @@ def create_user(
     user = FidesUser.create(db=db, data=user_data.dict())
     logger.info("Created user with id: '{}'.", user.id)
     FidesUserPermissions.create(
-        db=db, data={"user_id": user.id, "scopes": [PRIVACY_REQUEST_READ]}
+        db=db,
+        data={
+            "user_id": user.id,
+            "scopes": [PRIVACY_REQUEST_READ],
+        },  # TODO - Change this to Viewer Role by default?
```

Comment:
Currently when creating a user, they are given "privacy request read" scopes by default.  Once we have the UI for assigning roles in place, we might change the default to be a "viewer" role.